public/app/features/variables/constant/actions.test.ts

```diff
@@ -11,7 +11,7 @@ import { setCurrentVariableValue } from '../state/sharedReducer';
 import { initDashboardTemplating } from '../state/actions';
 
 describe('constant actions', () => {
-  variableAdapters.set('constant', createConstantVariableAdapter());
+  variableAdapters.setInit(() => [createConstantVariableAdapter()]);
```

Comment:
really like that all the definitions are now lazy :)

Reply:
Clicked the confused reaction instead of thumbs up, sorry for that